test/mbr_update_test.rb

```diff
@@ -28,6 +31,16 @@
       )
     end
 
+    context "on s390" do
+      before do
+        allow(Yast::Arch).to receive(:architecture).and_return("s390_64")
+      end
+
+      it "does nothing except returning true" do
+        expect(subject.run).to eq true
```

Comment:
so you are not against using single architecture context? Otherwise I do not mind to change it.